README.md

```diff
@@ -109,3 +109,10 @@ A commented example configuration::
     #
     # Best practice: make sure it ends in a /
     redirect_url_base = http://localhost:6543/data/
+
+    # this is the public host and scheme to reach the service
+    # like https://shavar.stage.mozaws.net
+    # when not provided, uses X-Forwarded-Host then fallsback to HTTP_HOST
+    # and X-Forwarded-Proto for the scheme
+    host = shavar.in.production.mozilla.com
+    scheme = https
```

Comment:
This block is added under the `[moz-abp-shavar]` section? Is it supposed to be under the `[server:main]` section?